### ISSUE TITLE
Pascal's triangle right rows

### ISSUE THREAD

**Issue by fedur** [snu-sf-class/pp201701#6]
Hello all, 
for problem 1, it says we have to add "upper left value once more" to calculate (r,c) but this special rule doesn't seem to apply to the values at the rightmost side of the triangle (when r==c) if I refer to the implementation of the picture in comment (which always shows a 1 when r==c).

Thus, do I need to apply the special rule to the rightmost Value (as in the text) or do I base my implementation on the picture and put all 1 when i'm on the right edge of the triangle?
Hope my question is clear ..!

Thank you!

Félix

**Comment by kim-yoonseung** [comment]
Yes. Like the drawing of the triangle in the problem, you need a special rule for the (c==0) and (c==r) cases.
